ztp/gitops-subscriptions/argocd/example/policygentemplates/group-du-sno-ranGen.yaml

```diff
@@ -58,14 +58,14 @@ spec:
           ptp4lOpts: "-2 -s --summary_interval -4"
           phc2sysOpts: "-a -r -n 24"
```

Comment:
I don't know if this change is in scope here, but the fact that this is a "group" template but the PTP config just above has a specific interface name seems dangerous to me.